### ISSUE TITLE
Failure to build module I2S kernel Module

### ISSUE THREAD

**Issue by phenixxmos** [xmos/vocalfusion-rpi-setup#67]
I was running a Raspberry pi 4 with 2g with running the lastest Raspberry Pi OS with kernel 5.15.84-v7l+. The './setup.sh xvf3800-intdev' failed with error

'The following package was automatically installed and is no longer required:
  libfuse2
Use 'sudo apt autoremove' to remove it.
0 upgraded, 0 newly installed, 0 to remove and 41 not upgraded.
make CFLAGS_MODULE='-DRPI_4B -DI2S_MASTER'
# copy the source file to avoid maintaining multiple copies inside the Makefile folders
# We tried to use KBUILD_OUTPUT but the module was not compiling
cp ../src/loader.c i2s_master_loader.c
make -C /lib/modules/5.15.84-v7l+/build M=/home/xmostest/vocalfusion-rpi-setup/loader/i2s_master modules
make[1]: *** /lib/modules/5.15.84-v7l+/build: No such file or directory.  Stop.
make: *** [Makefile:9: all] Error 2
Error: I2S kernel module build failed'

**Comment by xluciano** [comment]
@phenixxmos, which Raspbian version are you using?
Can you post the output of `cat /etc/os-release`?
The instructions in this repo should work with Bullseye.

I followed the same instructions at a similar time as when the issue was created and I got a different kernel version:

```
uname -a
Linux raspberrypi 6.1.19-v7l+ #1637 SMP Tue Mar 14 11:07:55 GMT 2023 armv7l GNU/Linux
```

**Comment by xluciano** [comment]
Fixed in v5.8.0
